qlib/pagetable.rs

```diff
@@ -41,17 +71,50 @@ cfg_aarch64! {
                                                          PageTableEntry, PageTableIndex,
                                                          PageTableFlags};
 
+   //
+   // A swapt-out page has the bit-flag set.
+   //
+   fn is_pte_swapped(flags: PageTableFlags) -> bool {
```

Comment:
please use the functions from bitflags! e.g. .contains(), .insert(), .remove().

The primitive bitwise ops are error prone

Reply:
Done.